CONTRIBUTING.md

```diff
@@ -22,4 +22,35 @@ We do all of NeMo-text-processing's development in the open. Contributions from
 11) Optional: if you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
 12) Send your PR and request a review
 
+# Notes for Language Contribution
+1) `en/graph_utils.py` is the de facto parent for all other `graph_util` functions. Please refrain from duplicating code logics into new `graph_utils.py` files and default to imports from `en/graph_utils.py` instead. `LANG/graph_utils.py` files should only contain new methods and variables. Not all new languages will require a submodule specific `graph_utils.py` file.
```

Comment:
do we want to do the same for "en/utils.py" or better not?

Reply:
I think they should default the `en/utils` since that works as a default class for the rest of the languages. A lot of time people just copy paste from that file into the `graph_utils` file of the respective language, which makes any downstream changes annoying to catch cause then you need to check per util. If they're all coming from one place it's easier to debug.

Reply:
I completely agree -- my question is whether we should mention both utils and graph_utils instead of just graph_utils

Reply:
oh misunderstood, mb. yeah let's also bring up `en/utils`